aad/spring-cloud-azure-starter-active-directory/aad-web-application-and-resource-server/src/main/java/com/azure/spring/sample/aad/security/AADWebApplicationAndResourceServerConfig.java

```diff
@@ -3,8 +3,8 @@
 
 package com.azure.spring.sample.aad.security;
 
-import com.azure.spring.cloud.autoconfigure.aad.AADResourceServerWebSecurityConfigurerAdapter;
```

Comment:
do we need change name AADWebApplicationAndResourceServerConfig to AadWebApplicationAndResourceServerConfig?

Reply:
> do we need change name AADWebApplicationAndResourceServerConfig to AadWebApplicationAndResourceServerConfig?

FYI:https://azure.github.io/azure-sdk/java_introduction.html#java-naming-uppercase-acronyms ,it may yes

Reply:
yes